buildSrc/src/main/kotlin/io/opentelemetry/gradle/ProtoFieldsWireHandler.kt

```diff
@@ -99,7 +99,10 @@ class ProtoFieldsWireHandler : SchemaHandler() {
       fun get(schema: Schema): JavaGenerator {
         val nameToJavaName = linkedMapOf<ProtoType, TypeName>()
         for (protoFile in schema.protoFiles) {
-           val javaPackage = javaPackage(protoFile)
+          if (protoFile.location.path == "wire/extensions.proto") {
```

Comment:
Looks like the latest version automatically includes a new proto file, which doesn't include a java package. This was causing the build to fail. Skipping this in the code gen because we don't have anything that references wire/extensions.proto